packages/childchain/childchain.sh

```diff
@@ -27,22 +28,14 @@ ethermintd version
 ethermintd init $MONIKER --chain-id $CHAINID
 
 # Allocate genesis accounts (cosmos formatted addresses)
-ethermintd add-genesis-account $(ethermintcli keys show $KEY -a) 100000000000000000000000aphoton,1000000000000000000stake
+ethermintd add-genesis-account $(ethermintcli keys show $KEY -a) 10000000000000001000000000000aphoton,1000000000000000000stake
```

Comment:
- instead of 10000000000000001000000000000, mint `0xff..30xff..ff` (max supply) to deployment account. 
- also give stake to other 3 validators here